posts/2024-07-12_teal_app_development_pharmaverseadam/teal-app-development.qmd

```diff
@@ -0,0 +1,208 @@
+---
+title: "Simplifying Clinical Data Dashboards with Teal and PharmaverseAdam"
+author:
+  - name: Pavel Demin
+  - name: APPSILON
+  - name: Dror Berel
+description: "Learn how to quickly develop interactive and reproducible clinical data dashboards using Teal and PharmaverseAdam." # <---- please write a description
+date: "2024-07-12"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [Technical, Community]
+# feel free to change the image
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-07-12_teal_app_development_pharmaverseadam"
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+Every developer must solve two difficult problems when creating a Shiny application (in fact, any application) from the ground up: software architecture and data design. In the world of clinical data analysis, however, much development has been aimed at providing a jump-start approach to creating R/Shiny applications that would take away most of the pain caused by these two problems.
+
+This blog should help you get an idea of how easy it is to get started with the [pharmaverse](https://pharmaverse.org/) ecosystem. We will create an interactive clinical data dashboard using `{teal}` and will use `{pharmaverseadam}` as the data source.
```

Comment:
can you link teal and pharmaverseadam - we have the links package to do this - see admiral 1.1.1 post

Reply:
Done!